packages/table/src/table.tsx

```diff
@@ -41,9 +41,9 @@ import { TableBody } from "./tableBody";
 export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     /**
      * If `false`, only a single region of a single column/row/cell may be
-     * selected at one time. Using <kbd class="pt-key">ctrl</kbd> or
-     * <kbd class="pt-key">meta</kbd> key will have no effect,
-     * and a mouse drag will select the current column/row/cell only.
+     * selected at one time. Using <code>ctrl</code> or <code>meta</code>
```

Comment:
why switch away from `.pt-key`?

but if you must, at least use backticks instead of writing out `<code>`. same with file above.